api/v1/tetragon/sensors.proto

```diff
@@ -140,13 +143,38 @@ enum LogLevel {
 	LOG_LEVEL_TRACE = 6;
 }
 
+message DumpProcessCacheArgs {
+	bool skipZeroRefCnt = 1;
+}
+
+message ProcessInternal {
+	Process process = 1;
+	string color = 2;
+	google.protobuf.UInt32Value refcnt = 3;
+	// refcntOps is a map of operations to refcnt change
+	// keys can be:
+	// - "process++": process increased refcnt (i.e. this process starts)
+	// - "process--": process decreased refcnt (i.e. this process exits)
+	// - "parent++": parent increased refcnt (i.e. a process starts that has this process as a parent)
+	// - "parent--": parent decreased refcnt (i.e. a process exits that has this process as a parent)
+	map<string, int32> refcntOps = 4;
+}
+
+message ProcessesInternal {
+	repeated ProcessInternal processes = 1;
+}
+
 message GetDebugRequest{
 	ConfigFlag flag = 1;
+	oneof arg {
+		DumpProcessCacheArgs dump = 2;
+	}
 }
 message GetDebugResponse{
 	ConfigFlag flag = 1;
 	oneof arg {
 		LogLevel level = 2;
+		ProcessesInternal processes = 3;
```

Comment:
nit: it a bit less clear which field is for which call between:
```
	CONFIG_FLAG_LOG_LEVEL = 0;
	CONFIG_FLAG_DUMP_PROCESS_CACHE = 1;
```
Since we don't put them in a struct like the args with `DumpProcessCacheArgs` for example.

Reply:
Done, renamed `ProcessesInternal` to `DumpProcessCacheResArgs`.